src/GraphTable.js

```diff
@@ -113,6 +96,13 @@ export default class GraphTable extends PureComponent {
         ]
       }
 
+      const sparklineData = item.data.map((d, i) => {
+        return {
+          change: d,
+          name: `${d.toFixed(3)}% - ${summary.dates[i]}`
```

Comment:
Since the tooltip automatically displays the `change` key, just make `name` the date.

Also, use a more appropriate name than `change` (it's not a change, just the value at that point in time). Maybe `Percent`? Make sure to truncate it to 3 decimal places as well (`parseFloat(d.toFixed(3))`)